### PR TITLE
Feat/stacks cli

### PR DESCRIPTION
**Pull request by guillermogutierrez**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

Support for the stacks cli v2.0 integration

#### 🤔 Why

With the new version of the stacks cli we need to adapt the code to the new requirements.

#### 🛠 How

Stacks cli expects a file called stackscli.yml file with all build steps. Also existing files require minor updates to ensure the scaffolding works as expected

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml [hunk 1]
```diff
@@ -170,7 +170,7 @@ variables:
   - name: java_project_type
     value: "Java App"
   - name: functional_test_project_type
-    value: "Functional API Tests"
+    value: "Functional API Tests2"
```
Comment (radoslaw-orlowski) [inline build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml:173]: Is it supposed to say Tests2?

**Review by radoslaw-orlowski — APPROVED** [review]

File: build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml (hunk 1 above)
Comment (guillermogutierrez) [reply build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml:173]: I added that to check the cli paorsin, but forget to revert it back.

**Pull request merged**